packages/now-python/src/index.ts

```diff
@@ -16,6 +16,14 @@ import {
 
 async function pipInstall(pipPath: string, workDir: string, ...args: string[]) {
   const target = '.';
+  // See: https://github.com/pypa/pip/issues/4222#issuecomment-417646535
```

Comment:
This linked issue seems to indicate the correct env var is `PIP_IGNORE_INSTALLED`, not `PIP_USER`

Reply:
the patch by default sets two parameters: `--user` and `--ignore-installed`. Their values can be overwritten by `PIP_USER` and `PIP_IGNORE_INSTALLED` environment variables, respectively.

The error is caused by the `--user` flag, so `PIP_USER` var should be used.